cmd/pipelineCreateScanSummary.go

```diff
@@ -60,7 +60,7 @@ func runPipelineCreateScanSummary(config *pipelineCreateScanSummaryOptions, tele
 
 	output := []byte{}
 	if len(config.PipelineLink) > 0 {
-		output = []byte(fmt.Sprintf("## Pipeline Source for Details\n\n[%v](%v)\n\n", config.PipelineLink, config.PipelineLink))
+		output = []byte(fmt.Sprintf("## Pipeline Source for Details (since listed results might be incomplete)\n\nIt is crucial that you follow this link to check the detailed pipeline status:\n\n[%v](%v)\n\n", config.PipelineLink, config.PipelineLink))
```

Comment:
I would inline the link to the pipeline and out the () in the headline:

```suggestion
		output = []byte(fmt.Sprintf("## Pipeline Source for Details\n\nAs listed results might be incomplete, it is crucial that you check the detailed [pipeline](%v) status.\n\n", config.PipelineLink))
```